.devcontainer/docker-compose.yml

```diff
@@ -30,6 +32,7 @@ services:
     secrets:
       - triggers
       - mqtt
+      - telegram
```

Comment:
This needs to be commented out before merge. The default developer configuration shouldn't have this enabled since there's no reasonable way to locally test/run it in Docker.

Reply:
Fixed